dev/com.ibm.ws.transport.http/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -84,6 +88,9 @@ http.remoteip.proxies.desc=A regular expression that defines trusted proxies.
 http.remoteip.useRemoteIpInAccessLog=Use remote IP in access log
 http.remoteip.useRemoteIpInAccessLog.desc=If this property is set to true and the remote client information was verified by the HTTP Channel, the NCSA access log reflects the Forwarded or X-Forwarded-* headers. These headers are reflected when one or more of the following items are recorded: the remote client IP, the host, or the request protocol.
 
+http.autocompression=Compression Options
+http.autocompression.desc=An element configured within the httpEndpoint element so that the associated HTTP channel is enabled to consider compressing response bodies based on the client's Accept-Encoding header.
```

Comment:
We should use http.compression now that we've changed the configuration element to `compression` vs `autoCompression`